src/main/java/com/bravo/user/model/filter/PaymentFilter.java

```diff
@@ -0,0 +1,14 @@
+package com.bravo.user.model.filter;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Combinine Builders and Constructors allows for 3 different ways to set the value, the builder, the constructor, and the setter.  Are all of these needed?

Reply:
I'm only using the builder for this class, so really not needed.   Habit from my last project of adding all 3 of these.